bin/compile

```diff
@@ -76,7 +76,7 @@ SBT_BINDIR="$SBT_USER_HOME"/bin
 cd $BUILD_DIR
 
 # unpack cache
-CACHED_DIRS="$SBT_USER_HOME target project/target project/boot .coursier"
+CACHED_DIRS="$SBT_USER_HOME target project/target project/boot .cache/coursier"
```

Comment:
I'm curious why `.coursier` was added here but not `.ivy2`. This would require sbt 1.3 or greater, which I don't think is clearly documented.

Reply:
I think the whole `.coursier` entry in `CACHED_DIRS` is no longer required. We do cache `$SBT_USER_HOME` which includes `.cache/coursier` as well as `.ivy2`. It might have been added prior to sbt adopting coursier which might not have respected our settings for the base sbt directory.